minari/dataset/minari_storage.py

```diff
@@ -118,11 +118,27 @@ def apply(
                     "rewards": ep_group["rewards"][()],
                     "terminations": ep_group["terminations"][()],
                     "truncations": ep_group["truncations"][()],
+                    "infos": self._decode_infos(ep_group["infos"])
+                    if "infos" in ep_group
+                    else {},
                 }
                 out.append(function(ep_dict))
 
         return out
 
+    def _decode_infos(self, infos: h5py.Group):
```

Comment:
To avoid the user having to define an "info space" the idea is to use whatever comes out of the first reset as the info template, and if need be, generate whatever manifest would be needed to save it to a particular back-end.

Reply:
Yes, I guess we will see in the future which solution is the best